### PR TITLE
show boosted apr

### PR DESCRIPTION
**Pull request by t0rbik**
## PR-Codex overview
This PR focuses on adding a new filter option and a link to the Options page in the LiquidityPairsTable component.

### Detailed summary
- Added a new filter option "boosted" to the filter list.
- Added a link to the Options page for pairs with a positive "oblotr_apr" value.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx [hunk 1]
```diff
@@ -220,25 +222,27 @@ const EnhancedTableToolbar = ({
...
+                  value={filterOption}
+                  checked={filter === filterOption}
+                  onChange={onChange}
+                  className="peer hidden"
+                />
+                <label
+                  htmlFor={filterOption}
+                  className="flex min-h-[56px] min-w-[108px] cursor-pointer items-center justify-center rounded-lg border border-[rgba(255,255,255,0.23)] px-2 font-medium transition-colors hover:bg-emerald-900 peer-checked:border-emerald-900 peer-checked:bg-background peer-checked:font-semibold peer-checked:text-lime-50"
+                >
+                  <div className="uppercase">{filterOption}</div>
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx:241]:
```suggestion
                  <span className="capitalize">{filterOption}</span>
```
Why not use `span`? And have you tried `capitalize`?

image

Much delicate IMO.

File: Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx [hunk 2]
```diff
@@ -855,6 +859,17 @@ function Row(props: { row: Pair; onView: (_row: Pair) => void }) {
                 <WarningOutlined className="ml-2 text-base text-yellow-300" />
               </Tooltip>
             )}
+            {row.oblotr_apr > 0 && (
+              <Tooltip
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx:863]: And style of this tooltip is sort of boring, hope we can make it better in near future.

File: Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx (hunk 1 above)
Comment (t0rbik) [reply Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx]: we can come up with something but i like uppercase the most in this case, and italic the least

File: Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx (hunk 2 above)
Comment (t0rbik) [reply Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx:863]: definitely

**Review by oxSaturn — APPROVED** [review]

**Pull request merged**